### DISCUSSION TITLE
Pockets Explorations

### DISCUSSION THREAD

**Discussion by deannagelosi** [UnstableDesign/AdaCAD-UI#185] [Show and tell]
@EttaSandry I'm learning how to weave pockets/double weave and would love some advice. Here's what I've done so far, what would you suggest I try next?
![IMG_5924](https://github.com/user-attachments/assets/fe9bc784-c324-4dee-9a06-543f634c1b87)

**Comment by Devendork** [comment]
Hi @deannagelosi can you upload your AdaCAD file?

**Comment by deannagelosi** [comment]
[pockets_explorations.json](https://github.com/user-attachments/files/17154017/pockets_explorations.json)

**Comment by EttaSandry** [comment]
This is looking great @deannagelosi ! I love the areas that combine some pocket with some not-pocket. You could try different structures in the pocket and ground areas, pockets with more than two layers, or pockets all the way across with no ground in between.
